Topic/fridge sprite

EveDiby opened the pull request:
There is now a fridge for cell samples. You can control the fridge temperature by clicking on the fridge and entering the temperature you would like to set the fridge and ultimately the cell samples to.

JRCockrum reviewed (changes requested):
The fridge looks good but it should be moved somewhere else in the scene since right now it is directly in front of the microscope. 

currently when you open the fridge you don't see the temperature it is currently set to. You have to enter a new value to see the current temperature setting

Also I think that the temperature control could be made a bit simpler, instead of having a keypad it might be easier to just have two buttons to increase and decrease the temperature.  That way we can have set range of temperatures and we dont have to worry about handling edge cases for user input (for example if the user sets the temperature on the keypad to be 99999999)

Drock54651 commented:
I think the keypad is fine if that accurately represents the actual fridge. For the edge cases, I do think it could potentially be left in as another potential user error.

EveDiby commented:
I moved the fridge over and added constraints so that the only valid range of temperatures a user can type in are between 0 and 50 degrees. Also, when the fridge is first opened, by default it is set to 32 degrees and is written as such for users to see.

JRCockrum reviewed (approved):
Looks good!
